website/docs/Pipelines/RNA_with_UMIs_Pipeline/README.md

```diff
@@ -0,0 +1,203 @@
+
+#### 6. Metric calculation
+
+The pipeline uses [RNA-SeQC](https://github.com/getzlab/rnaseqc), Picard’s [CollectRNASeqMetrics](https://gatk.broadinstitute.org/hc/en-us/articles/360037057492-CollectRnaSeqMetrics-Picard-), and Picard’s [CollectMultipleMetrics](https://gatk.broadinstitute.org/hc/en-us/articles/360037594031-CollectMultipleMetrics-Picard-) to calculate summary metrics that can be used to assess the quality of the data each time the pipeline is run. The resulting metrics are output to several individual files and merged together to create the `MergeMetrics.unified_metrics` output file. 
+
+#### 7. Outputs
+
+Workflow outputs are described in the table below. 
+<!--- Link to metrics overview will go here --->
+
+| Output variable name | Description | Type | 
+| ------ | ------ | ------ |
+| transcriptome_bam | Duplicate-marked BAM file containing alignments from STAR translated into transcriptome coordinates. | BAM |
+| transcriptome_bam_index | Index file for the transcriptome_bam output. | BAM Index |
+| transcriptome_duplicate_metrics | File containing duplication metrics. | TXT | 
+| output_bam | Duplicate-marked BAM file containing alignments from STAR translated into genome coordinates. | BAM |
+| output_bam_index | Index file for the output_bam output. | BAM Index |
+| duplicate_metrics | Duplicate metrics file containing the number of reads marked as duplicates. | TXT |
+| rnaseqc2_gene_tpm | File containing TPMs. | GCT |
+| rnaseqc2_gene_counts | File containing gene counts. | GCT |
+| rnaseqc2_exon_counts | File containing exon counts. | GCT |
+| rnaseqc2_fragment_size_histogram | File containing counts of observed fragment size. | TXT |
+| rnaseqc2_metrics | File containing RNA-SeQC metrics including strand specificity, 3’/5’ bias, rRNA reads, and others | TSV |
+| picard_rna_metrics | Metrics file containing the output of Picard’s CollectRnaSeqMetrics tool. | TXT |
+| picard_alignment_summary_metrics | Metrics file containing output of Picard’s CollectAlignmentSummaryMetrics tool. | TXT |
+| picard_insert_size_metrics | Metrics file containing output of Picard’s CollectInsertSizeMetrics tool. | TXT |
+| picard_insert_size_histogram | Histogram chart of insert size. | PDF |
+| picard_base_distribution_by_cycle_metrics | Metrics file containing the output of Picard’s CollectBaseDistributionByCycle tool. | TXT |
+| picard_base_distribution_by_cycle_pdf | Chart of nucleotide distribution per cycle. | PDF |
+| picard_quality_by_cycle_metrics | Metrics file containing the output of Picard’s MeanQualityByCycle tool. | TXT |
+| picard_quality_by_cycle_pdf | Chart of mean quality by cycle. | PDF |
+| picard_quality_distribution_metrics | Metrics file containing the output of Picard’s QualityScoreDistribution tool.
+| unified_metrics | Merged metrics file containing RNA-SeQC, Picard RNA-seq, Picard insert size, and Picard alignment metrics. | TXT |
```

Comment:
unified_metrics has been moved to BroadInternalRNAwithUMIs.

Reply:
updated